### ISSUE TITLE
Error uploading a layer in geonode 3.2x

### ISSUE THREAD

**Issue by Gim88** [GeoNode/geonode#7915]
## Expected Behavior
Error uploading a layer in geonode 3.2.x

## Actual Behavior
Unexpected error! - Unknwon Error Code. Contact the system administrator for more information regarding this error message. Additional info: [The configured path for the file-system backend does not work correctly, please make sure that it exists and has the correct permissions.]

## Steps to Reproduce the Problem

  1. Installed geonode 3.2.x following the instructions in the manual
  2.  After that i uploaded a layer to test the system.
  3. And then a error had occurred while uploading the layer.
![geonode_upload_problem1](https://user-images.githubusercontent.com/88358056/127947642-93fd13d6-d6a8-40b0-80fc-e4deb081d7a3.png)

## Specifications

  - GeoNode version: 3.2.x
  - Installation method (manual):
  - Platform: Ubuntu 20.04
  - Additional details: I successfully installed the geonode 3.2.0 version using the instructions in the manual. https://docs.geonode.org/en/3.x/install/advanced/index.html

**Comment by t-book** [comment]
This normally happens with a configuration problem between geonode and geoserver. geoserver user credentials are a common problem.

**Comment by Gim88** [comment]
Thank you for the reply. I log in to GeoServer and check OAuth Client ID/ Client Secret key. I found out from the Client ID a statement followed by the ID Errno 2] No such file or directory: 'spatialite'. 

![client id](https://user-images.githubusercontent.com/88358056/128107970-399e51fe-45c8-445e-9cc0-3a41e834ab66.png)

I copied the Geonode OAuth ClientID/Client Secret key and paste it on the Geoserver OAuth. And then i restart the nginx and tomcat service to be sure. 

After that i log on to geonode and uploaded the same layer. Still, i encounter the same error. I logon to geoserver and check if the layer is uploaded in the system and i found out that is already uploaded to the geoserver.

![uploaded](https://user-images.githubusercontent.com/88358056/128108481-43d6eb16-b3b5-4aa0-bafb-89370652e1ea.png)

For your information. As for the installation of the geonode i skip installing spatialite. And make use postgis as my default database.
